coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/HiveSqlConformance.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2021 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.hive.hive2rel;
+
+import org.apache.calcite.sql.validate.SqlConformance;
+import org.apache.calcite.sql.validate.SqlDelegatingConformance;
+
+
+public class HiveSqlConformance extends SqlDelegatingConformance {
+
+  public static SqlConformance HIVE_SQL = new HiveSqlConformance();
+
+  private HiveSqlConformance() {
+    super(PRAGMATIC_2003);
+  }
+
+  @Override
+  public boolean allowNiladicParentheses() {
+    return true;
+  }
```

Comment:
Thanks for this change. @kxu1026 was about to publish a PR with a similar class (`HiveSqlConformance`). @kxu1026, please take a look and see if you would like to unify with this approach.